### ISSUE TITLE
Firefox Android scrolling issue

### ISSUE THREAD

**Issue by sixohsix** [BanchanArt/banchan#679]
Artist offering pages don't scroll properly on Firefox for Android. For example:

https://banchan.art/studios/sunsetmartiniart/offerings/full-body

This page can only be scrolled from the Banchan Art top banner. The main page doesn't scroll.

**Comment by zkat** [comment]
@sixohsix a fix for something like this was recently deployed. Can you double-check and confirm this is still happening?

**Comment by sixohsix** [comment]
Since yesterday it seems *almost* fixed. The first press and scroll gesture doesn't work but all subsequent scroll gestures work. No idea why the first one is weird but it's not blocking me anymore. I'd call it fixed enough. 😸

**Comment by zkat** [comment]
I'll keep this open until one of us can test things on Android (I'm on iOS). Even if it only happens sometimes, that's not great behavior.

**Comment by skullbunnygalaxy** [comment]
I cannot replicate on Firefox on my Pixel 6. 

@sixohsix Is it giving you issues when pressing on a specific part of the page?

**Comment by sixohsix** [comment]
For me, the first scroll gesture on any part of the page fails. I can make the Firefox URL Footer show and hide, but the page doesn't scroll. All subsequent scroll interactions after the first one are fine.

Firefox 117.0 (Build #2015969571), e181aecfa7+
GV: 117.0-20230821145345
AS: 117.0

On a Xiaomi Mi 11 lite running Android 13.

**Comment by skullbunnygalaxy** [comment]
@sixohsix Do you have any add-ons installed on Firefox mobile?

**Comment by sixohsix** [comment]
Yes, but I disabled all my add-ons, restarted Firefox and refreshed the page. The issue is the same as before.
